crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -2061,7 +2078,8 @@ mod tests {
         let info: RoomInfo = serde_json::from_value(info_json).unwrap();
 
         assert_eq!(info.room_id, room_id!("!gda78o:server.tld"));
-        assert_eq!(info.room_state, RoomState::Invited);
```

Comment:
I'm not a big fan of importing enum variants like this. Is this something you've seen elsewhere in the SDK? If not, please can you change these back to e.g. `RoomState::Joined` ?

Reply:
Also, in these test `Invited` was already imported directly without the `RoomState` prefix so I just followed the convention here. I'll revert it though, being more explicit seems less prone to errors.